content/jobs.md

```diff
@@ -5,35 +5,54 @@ aliases = [ "node/785" ]
 
 The ReactOS Project is seeking qualified individuals to fill multiple positions. Contracted **full time, part time, and scholarship (internship)** level positions are all available.
 
+If you do qualify for a position, you'll be joining the very small and exclusive ranks of people who know how to do NT systems development. And believe us, those ranks are very, very small. And yes, you will be gaining an immense amount of knowledge working with such people. 
```

Comment:
https://jira.reactos.org/browse/ONLINE-822 states that we want to make us seem less elitist.
This sentence goes directly against that.

Reply:
We could maybe change the "seeking qualified individuals" to "seeking individuals" and

" If you do qualify for a position, you'll be joining the very small and exclusive ranks of people who know how to do NT systems development. And believe us, those ranks are very, very small. And yes, you will be gaining an immense amount of knowledge working with such people. "

to

"If you qualify for a position, you'll be joining the very small and exclusive ranks of people who know how to do NT systems development. And believe us, those ranks are very, very small so you will be gaining an immense amount of knowledge working with such people. It's a great way for beginners to interact with people who have been working in this field for over the years."

This way it looks more beginner-friendly.